resources/windows/wix/condition.wxs.erb

```diff
@@ -18,11 +18,10 @@
         <![CDATA[Installed OR (VersionNT >= 600)]]>
       </Condition>
 
-      <!--
       <Condition Message="32 bit <%= settings[:product_name] %> is no longer supported on 64 bit Windows Operating Systems.">
-          <![CDATA[Installed OR VersionNT64]]>
+          <![CDATA[Installed OR Not VersionNT64]]>
```

Comment:
Hrmmm @glennsarti Trying to work out a context where this (```Installed```) would be useful if we are specifically looking to block it going onto a 32 bit box.